sys/net/gnrc/network_layer/ndp/gnrc_ndp.c

```diff
@@ -547,7 +547,13 @@ void gnrc_ndp_rtr_adv_handle(kernel_pid_t iface, gnrc_pktsnip_t *pkt, ipv6_hdr_t
                 break;
 #endif
         }
-        sicmpv6_size -= (opt->len * 8);
+
+        uint32_t opt_length = (opt->len * 8);
+        sicmpv6_size -= opt_length;
+        opt_offset += opt_length;
```

Comment:
Can you please use the same format as with the other message handle functions (e.g. `nbr_adv_handle`).